packages/icons/icons.json

```
@@ -1430,49 +1430,49 @@
     {
         "displayName": "Stop",
         "iconName": "stop",
-        "tags": "player, media, controls, digital, analogue, film, audio, video",
+        "tags": "player, media, controls, digital, analogue, film, audio, video, box, square, error, not possible, blocked, black square, shape",
```

```suggestion
        "tags": "player, media, controls, digital, analogue, film, audio, video, box, square, error, not possible, blocked, shape, filled",
```

see my other comment about shape color "black"